components/license-control-service/pkg/storage/storage.go

```diff
@@ -0,0 +1,261 @@
+package storage
```

Comment:
This new package makes it a bit easier to inject a different storage system.  I added this when I though I might just do unit tests without pg, but then I integrated pg into the unit tests anyway.  However, I like the isolation this gives still.